code/modules/vehicles/scooter.dm

```diff
@@ -214,33 +209,35 @@
 	return
 
 /obj/vehicle/ridden/scooter/skateboard/improvised/attackby(obj/item/I, mob/user, params)
-	if(istype(I, /obj/item/stack/rods))
-		if(!I.tool_start_check(user, amount=2))
-			return
-		to_chat(user, "<span class='notice'>You begin making handlebars for [src].</span>")
-		if(I.use_tool(src, user, 25, volume=50, amount=2))
-			to_chat(user, "<span class='notice'>You add the rods to [src], creating handlebars.</span>")
-			var/obj/vehicle/ridden/scooter/S = new(loc)
-			if(has_buckled_mobs())
-				var/mob/living/carbon/H = buckled_mobs[1]
-				unbuckle_mob(H)
-				S.buckle_mob(H)
-			qdel(src)
-	else
+	if(!istype(I, /obj/item/stack/rods))
 		return ..()
+	if(!I.tool_start_check(user, amount=2))
+		return
+	to_chat(user, "<span class='notice'>You begin making handlebars for [src].</span>")
+	if(!I.use_tool(src, user, 25, volume=50, amount=2))
+		return
+	to_chat(user, "<span class='notice'>You add the rods to [src], creating handlebars.</span>")
+	var/obj/vehicle/ridden/scooter/S = new(loc)
+	if(has_buckled_mobs())
+		var/mob/living/carbon/H = buckled_mobs[1]
+		unbuckle_mob(H)
+		S.buckle_mob(H)
```

Comment:
Could you make these better?

Reply:
to be clear, I was referring to var names

Reply:
Done for both https://github.com/tgstation/tgstation/pull/57626/commits/49336ce06b2aec3b226c40126fe39a4bb0f3afac